plugins/laravelvapor/vapor.go

```diff
@@ -0,0 +1,27 @@
+package laravelvapor
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func LaravelVaporCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Laravel Vapor CLI",
+		Runs:    []string{"vapor"},
```

Comment:
From what I could find in the docs, the calls to this CLI are usually made with 
```
php vendor/bin/vapor <command>
```
i.e. `vapor` is already a shell alias that has to be set by the user, is this correct?

In this situation, would it be worth looking into indicating that the command runs `php vendor/bin/vapor` instead? I think this would be nice, in order to cover all the different setups that users of this CLI might have.

Reply:
Unlike the Forge CLI, the Vapor CLI can be installed both globally and on a per-project basis.

The global installation follows the same pattern as described here: https://github.com/1Password/shell-plugins/pull/244#discussion_r1166753444, which means the Vapor CLI is available as `vapor`

My best guess is that the globally installed method is more popular as it offers mostly a one-time setup. (in my opinion)

As for the per-project installation method:
`vapor` as a shell alias is only applicable if the user chooses to make it an alias (it's not mandatory) and refers specifically to the per-project installation method.
But even if the user chooses to make it a shell alias, would that conflict with the `.op/plugins.sh` aliases?
Maybe a note in this plugin's docs could advise the user to make the alias for a consistent experience?

Another approach that crosses my mind would be for the plugin to handle both if there is a way to specify it in the `vapor.go` file or somewhere:
```
Runs: []string{"vapor", "php vendor/bin/vapor "},
```
Not sure if that's possible or even what's the proper syntax to specify more than one item 😄 

Not sure what's the best option here to handle both cases, but I would appreciate any feedback.

Reply:
So let me make sure I understand:

`vapor` and `php vendor/bin/vapor` are mutually exclusive ways of calling the binary: the first relates to the global installation, the second, to the project-scoped installation?

Reply:
In this case, then, I think we should indeed default to only supporting `vapor`, at this stage, to validate that only instances of global installation can use the plugin.

Shell Plugins currently don't have native support for project-scoped executables, but that is an interesting use-case!

Reply:
> So let me make sure I understand:
> 
> 
> 
> `vapor` and `php vendor/bin/vapor` are mutually exclusive ways of calling the binary: the first relates to the global installation, the second, to the project-scoped installation?

Absolutely correct 👍

Reply:
> In this case, then, I think we should indeed default to only supporting `vapor`, at this stage, to validate that only instances of global installation can use the plugin.
> 
> 
> 
> Shell Plugins currently don't have native support for project-scoped executables, but that is an interesting use-case!

Sounds good to me 😊 

In the future if project-scoped executables become natively supported I'll probably try to add it as a feature enhancement to this plugin 🤝